application/Library/Crypto.php

```diff
@@ -40,6 +40,14 @@ protected static function getKey() {
 		return self::$key;
 	}
 
+	private static function doWeNeedHiddenStrings() : bool {
```

Comment:
Maybe follow the existing function declaration style for consistency

Reply:
You mean because of the return type declaration ”: bool“?

Reply:
yes.

Reply:
This is new to PHP7.0. If we're ready to drop PHP5.6 support from now on, I think this improves code quality.

Reply:
yes you are right... it improves, just that it looks weird now to be used only in one method in a the whole class.

Reply:
True, but you have to start somewhere ;). If you don't like it, we can remove it. Anyway, as you wrote below, we should test this first.

Reply:
I just noticed, if we drop PHP7 support, this is obligatory anyway and we don't need to check for it. If we keep PHP5.6 support, this code doesn't run. So the way I created the PR doesn't make sense in any case.

Reply:
I think @rubenarslan should have the final call on dropping PHP5.6 support. I'd support this decision, as we're not aware of any formr instances running on such an outdated system at the moment and no one will start a new instance on such an outdated version.

Reply:
I'm pro. Let's drop it, before someone decides to run formr on 5.6 ;-)

Reply:
Is has been dropped in the documentation now. So if you agree, we can start implementing new functions with predefined return types.